### PR TITLE
Implemented logic of sorting chart data by date range.

### PR DESCRIPTION
**Pull request by sergtm1**

### CODE REVIEW

File: src/main/java/com/sergtm/service/impl/HeartRateServiceImpl.java [hunk 1]
```diff
@@ -95,36 +96,50 @@ public void addHeartRateById(Long id, int upperPressure, int lowerPressure, int
...
     @Override
-    public Collection<StatisticOnDay> getChartData(Long personId) {
+    public Collection<StatisticOnDay> getChartData(Long personId, String from, String to) {
         LocalDate now = LocalDate.now();
+        LocalDateTime fromDate;
+        LocalDateTime toDate;
         LocalDateTime firstDayOfMonth = now.withDayOfMonth(1).atStartOfDay();
         LocalDateTime lastDayOfMonth = now.withDayOfMonth(now.lengthOfMonth()).atTime(23, 59);
 
+        if (from == null || from.equals("")) {
```
Comment (aleksey-timofeev) [inline src/main/java/com/sergtm/service/impl/HeartRateServiceImpl.java]: Use StringUtils.isEmpty() function

File: src/main/java/com/sergtm/service/impl/HeartRateServiceImpl.java [hunk 2]
```diff
@@ -95,36 +96,50 @@ public void addHeartRateById(Long id, int upperPressure, int lowerPressure, int
...
+            fromDate = firstDayOfMonth;
+        } else {
+            fromDate = LocalDateTime.parse(from);
+        }
+
+        if (to == null || to.equals("")) {
```
Comment (aleksey-timofeev) [inline src/main/java/com/sergtm/service/impl/HeartRateServiceImpl.java]: Create a function parseDate with default value. Something like that:
public  Date parseDate(Date parsed, Date defaultValue)

Move it to static DateUtil class

Comment (sergtm1) [reply src/main/java/com/sergtm/service/impl/HeartRateServiceImpl.java]: Done

File: src/main/java/com/sergtm/service/impl/HeartRateServiceImpl.java (hunk 1 above)
Comment (sergtm1) [reply src/main/java/com/sergtm/service/impl/HeartRateServiceImpl.java]: Done

**Pull request merged**